about.html

```diff
@@ -4,9 +4,97 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="./styles.css">
     <title>Document</title>
 </head>
 <body>
-    
+    <section class="about">
+        <header class="head">
+            <div class="top">
+                <a href="#"><i class="fa fa-facebook-f"></i></a>
+                <a href="#"><i class="fa fa-twitter"></i></a>
+                <span>English</span>
+                <span>My page</span>
+                <span>Logout</span>
+            </div>
+            <nav class="nav-bar">
+                <a href="./index.html">
+                    <img src="./assets/Logo.png" alt="logo">
+                </a>
+                <ul class="nav-content">
+                    <li><a id="about" href="./about.html">About</a></li>
+                    <li>Program</li>
+                    <li>Join</li>
+                    <li>Sponser</li>
+                    <li>News</li>
+                    <li id="cc">CC Campaign</li>
+                </ul>
+            </nav>
+        </header>
+        <div class="main-page">
+            <div class="menu">
+                <a href="./index.html">Home</a>
+                <a href="./about.html">About</a>
+            </div>
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+            <div class="about-content">
+                <h1>"Hello! IT Lovers"</h1>
+                <h2>IT Live Annual Confrence 2022</h2>
+                <div class="about-contact">
+                    <p class="eventdesc">Eventfull conference that brings together tech indusry experts, companies and intersted indivsuals to share latest tech, future projects and business plans</p>
+                    <p class="contact">Please contact us per email for any further questions about IT live annual conference</p>
+                    <a href="#">itliveiraq@itliveorg.com</a>
+                </div>
+            </div>
+    </section>
+    <section class="logo">
+        <h2>The IT Live Anuual Conference 2022 Logo</h2>
+        <hr>
+        <p>The logo the IT live anuual conference 2022 was decided through the logo compitition from 8.June to 7.July</p>
+        <div class="logo-part">
+            <img src="./assets/Logo1.png" alt="">
+            
+        </div>
+        <hr id="logo-hr">
+    </section>
+    <section class="logo">
+        <h2>See the past IT live anuual conference</h2>
+        <hr>
+        <p>Take a look at the last two IT live anuual conferences in Baghdad and Irbil</p>
+        <div class="firstconf past">
+            <h2>2021</h2>
+            <p>IT live anuual conferences in Baghdad</p>
+        </div>
+        <div class="secondconf past">
+            <h2>2020</h2>
+            <p>IT live anuual conferences in Irbil</p>
+        </div>
+    </section>
```

Comment:
- [ ] Nice job working on these images! 💯 As shown in the given design, please make sure that in the desktop version, they're displayed horizontally:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/205169107-17725015-4203-420c-b30f-aa72a4c2a65d.png) | ![image](https://user-images.githubusercontent.com/77069011/205169004-61bb091e-0e8f-4139-b8fc-ccd628843652.png) |

Please consider fixing this! 🤓 🛠️